quadcopter_brain/src/quadcopter_brain/position_tools.py

```diff
@@ -27,13 +27,13 @@ def metered_offset(lat, lon, dX, dY):
         given_utm = geodesy.utm.fromLatLong(lat, lon)
         given_utm.easting += dX
         given_utm.northing += dY
-        return_lat_lon = given_utm.toMsg()
-        return return_lat_lon.latitude, return_lat_lon.longitude
+        return_lat_long = given_utm.toMsg()
```

Comment:
lat_long_diff? return doesn't tell me what it is, just that it's being returned, which I can see on the next line.